src/components/Socials/styles.scss

```
@@ -1,3 +1,64 @@
+// Mobile portrait
 .wrapper {
+	display: flex;
+	justify-content: center;
+	width: 100%;
 	font-size: 1em; //nothing comment to make sure class appears
+	
+		li {
+			width: 1em;
+			height: 1em;
+			list-style: none;
+		}
+
+			a {
+				display: block;
+				width: 100%;
+				height: 100%;
+				background-color: blue;
+
+				&[aria-label="Facebook"]{
```

```suggestion
				&.facebook{
```
Neat idea, but this would break if, for example, we find that "Visit us on Facebook" works better as a label for engagement or SEO or whatever; this fits under the category of 'content', I reckon, and we want to avoid tying styling to content where we can.

---

Need some advice on how to do className={ s.<insertsocialname>}. Sent you a message separately